open-reac/src/test/java/com/powsybl/openreac/OpenReacRunnerTest.java

```diff
@@ -259,39 +262,39 @@ void testRunAsync() throws IOException {
         }
     }
 
-    private void testAllModifAndLoadFlow(Network network, String subFolder, OpenReacParameters parameters) throws IOException {
-        runAndApplyAllModifications(network, subFolder, parameters, true);
+    private void testAllModifAndLoadFlow(Network network, String subFolder, OpenReacParameters parameters, ReportNode reportNode) throws IOException {
+        runAndApplyAllModifications(network, subFolder, parameters, true, reportNode);
         LoadFlowResult loadFlowResult = LoadFlow.run(network);
-        assertTrue(loadFlowResult.isOk());
+        assertTrue(loadFlowResult.isFullyConverged() || loadFlowResult.isPartiallyConverged());
```

Comment:
I saw that `isOk()` is deprecated. But is that function equivalent to `isFullyConverged() || isPartiallyConverged()`?

Reply:
I have changed this back to call isOk(), even if it is deprecated (isOk() is still called in powsybl-core ...)
Is this acceptable ?

Reply:
Replaced by a call to isFullyConverged (same was previously done in powsybl-open-loadflow)